### ISSUE TITLE
Android Accessibility - Master ticket (has many in it)

### ISSUE THREAD

**Issue by RaulConstan** [bcgov/bc-wallet-mobile#3257]
### Description of problem
This ticket contains many accessibility issues found during testing.  All issues found on Samsung S21 FE 5G (Android 13, 5.1 One UI)

1. Normally, you can scroll down to show the continue button, but in this case alternate choices would work: hide keyboard, or using a grid.

<img width="461" height="1128" alt="Image" src="https://github.com/user-attachments/assets/7266193b-a818-496d-91ef-24078fc5be55" />

2. Aside from the rollers not working with voice commands, showing and hiding the grid is inconsistent (the numbers change).

<img width="461" height="1128" alt="Image" src="https://github.com/user-attachments/assets/fcb51469-2b9a-4def-bed7-c83a9071fb68" />

3. Missing command 7

<img width="461" height="1128" alt="Image" src="https://github.com/user-attachments/assets/57fc05da-db5c-4400-a02f-45737b5dec8e" />

4. Missing command 7

<img width="461" height="1128" alt="Image" src="https://github.com/user-attachments/assets/760ecc63-1035-4c3a-88af-48b320f5a363" />

5. Missing command

<img width="450" height="1128" alt="Image" src="https://github.com/user-attachments/assets/346bf9e6-e9e5-4388-aacd-95df027f8336" />

6. Whenever the "Voice Access shortcut" is enabled, anything behind the button does not show the command number.

<img width="450" height="1128" alt="Image" src="https://github.com/user-attachments/assets/6878e752-b802-463d-be3b-e052bc5a5030" />

**Comment by jleach** [comment]
Giving this a 5 because iOS was estimated at a 5. I don't think both will take 10 but, maybe 6-8 but we'll just live with this one being sand-bagged.

**Comment by jleach** [comment]
@RaulConstan This appears to be an Android limitation — Voice Access favours showing numbers over labels by default. You can say "show numbers", "hide numbers", "show labels", and "hide labels" to toggle the overlays, but in most cases labels won't display the way they do on iOS.

For reference, I tested the v3 production setup on my Pixel 7a and it also shows numbers in most cases. Video attached. I think we should close this as either being reasonably close to v3 or an android limitation.

https://github.com/user-attachments/assets/28c1cabb-657b-424f-a7b5-ab47bad29a75

@cvarjao @hannahgov FYI.

**Comment by RaulConstan** [comment]
@jleach pull me into the discussion if needed.

**Comment by RaulConstan** [comment]
It's not perfect, as discussed with Jason, but it's better than it was.  Closing for now.
